flink-connectors/flink-connector-files/src/main/java/org/apache/flink/connector/file/src/AbstractFileSource.java

```diff
@@ -75,12 +75,12 @@
 
 	private final FileEnumerator.Provider enumeratorFactory;
 
-	private final FileSplitAssigner.Provider assignerFactory;
+	protected final FileSplitAssigner.Provider assignerFactory;
```

Comment:
I have a slight preference to keep these fields `private` and add a public getter (`public FileSplitAssigner.Provider getAssigner()`.

Reply:
+1 Some communities directly open mandatory checkstyle checks, providing only setters and getters instead of protected fields.